observability/build.gradle

```diff
@@ -7,7 +7,15 @@ plugins {
 
 description = "gRPC: Observability"
 dependencies {
-    api project(':grpc-api')
+    api project(':grpc-api'),
+            project(':grpc-netty-shaded')
```

Comment:
I think we don't need to add this dependency here. The application will choose whatever they want.

Reply:
Without this dependency I get the following error:

```
io.grpc.observability.LoggingChannelProviderTest > forTarget_interceptorCalled FAILED
    io.grpc.ManagedChannelProvider$ProviderNotFoundException: No functional channel service provider found. Try adding a dependency on the grpc-okhttp, grpc-netty, or grpc-netty-shaded artifact
```

For my testing at least I need netty-shaded. Do you agreee?

Reply:
Yes. It can be a `testImplementation` dependency.